.github/workflows/integration.yml

```diff
@@ -179,6 +200,13 @@ jobs:
           serve -m test::Python=file:///opt/ml/model/resnet18_serving_all.zip
           ./test_client.sh http://127.0.0.1:8080/predictions/test image/jpg https://resources.djl.ai/images/kitten.jpg
           docker rm -f $(docker ps -aq)
+      - name: Test TensorRT binary mode
```

Comment:
TensorRT won't work here. It requires GPU and has to be run in TensorRT docker image.